internal/cli/config.go

```diff
@@ -18,7 +18,8 @@ var (
 
 // Config is botkube cli config
 type Config struct {
-	Token string `json:"token"`
+	Token    string `json:"token"`
+	Identity string `json:"identity"`
```

Comment:
**[TO DISCUSS]**
I think it would be great to add the analytics property to config so that people won't need to disable telemetry every time. What do you think?

and maybe have a command like `botkube config set analytics=false` or something like this? 🤔 it could work in generic way, so someone could also execute `botkube config set token=...` manually 🤔 

Let's ask @mszostok what he thinks about this approach 👍